Microsoft.WindowsAzure.Storage/src/cloud_blob.cpp

```diff
@@ -1021,6 +1021,7 @@ namespace azure { namespace storage {
             *snapshot.m_metadata = *resulting_metadata;
             snapshot.m_properties->copy_from_root(*properties);
             snapshot.m_properties->update_etag_and_last_modified(protocol::blob_response_parsers::parse_blob_properties(response));
+            properties->update_etag_and_last_modified(protocol::blob_response_parsers::parse_blob_properties(response));
```

Comment:
What's it for?

Reply:
update the version id

Reply:
https://github.com/Azure/azure-storage-cpp/blob/c970b01af4918edc1cc6beb576837b38585abcbd/Microsoft.WindowsAzure.Storage/src/cloud_blob_shared.cpp#L35

update the version id